bigtable/client/filters_test.cc

```diff
@@ -193,3 +193,97 @@ TEST(FiltersTest, ApplyLabelTransformer) {
   auto proto = bigtable::Filter::ApplyLabelTransformer("foo").as_proto();
   EXPECT_EQ("foo", proto.apply_label_transformer());
 }
+
+/// @test Verify that bigtable::Filter::Condition works as expected.
```

Comment:
Code formatting for `bigtable::Filter::Condition` here and below?

Is it correct to assume that, when extracted, backticks turn to `<code>` tags or similar formatting? Or does Doxygen need something like Javadoc's `@code` annotation?

Reply:
Done.

> Is it correct to assume that, when extracted, backticks turn to <code> tags or similar formatting?

Yes.

 > Or does Doxygen need something like Javadoc's @code annotation?

It can do `@code/@endcode` too:

http://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdcode

In addition, we have turned on Markdown support in Doxygen, partly so the `README.md` file is included in the output.  That means things like block quotes (` ``` `) work.

Oh, and notice that the test list (those `@test` commands) is disabled, in the default `Doxyfile` because that is intended to generate the public documentation.